language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/ForeachCompletionUtil.java

```diff
@@ -183,8 +185,15 @@ public static String getTypeOfIteratorVariable(BallerinaCompletionContext ctx, T
         TypeSymbol rawType = CommonUtil.getRawType(symbol);
         switch (rawType.typeKind()) {
             case STRING:
+                type = CommonUtil.getModifiedTypeName(ctx, rawType) + ":Char";
```

Comment:
For this, do we need to call the `getModifiedTypeName` since this is straight forward? What if we use `string:Char` directly?